### PR TITLE
Bookstore: Styling

### PR DESCRIPTION
**Pull request by Santiago220991**
# Tasks done:

- Added styling to the project.
- Added animation to the book containers.
- Added pointer cursor to `Button` tags.
- Added hover effect to Add Book button.

### CODE REVIEW

File: src/components/AddBook.css [hunk 1]
```diff
@@ -41,3 +41,11 @@ form {
   width: 11.5rem;
   cursor: pointer;
 }
+
+#add:hover {
+  background-color: #1572b9;
+}
+
+#add:active {
+  background-color: #0290ff;
```
Comment (youmari) [inline src/components/AddBook.css]: - [ ] Please in order to follow the best practices of CSS you should not use `id` selectors  please try to change all the id selectors in your project to `Class` selectors 
image

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @Santiago220991 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Git flow
✔️ Good commit messages
✔️ Great title and short summary 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/AddBook.css (hunk 1 above)
Comment (Santiago220991) [reply src/components/AddBook.css]: Thanks.

**Comment by Santiago220991** [comment]
# Task done:

- Changed all the `id` selectors used to give styling for `class` selectors. 

**Note: There are still `id` selectors that are used by the code.**

**Review by omar-muhamad — APPROVED** [review]
# Approved ✔
### Hi @Santiago220991,

Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

thumbs-up-agree


## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by Santiago220991** [comment]
Thanks 👍

**Pull request merged**